texttospeech/test/quickstart.test.mjs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs/promises';
+import { existsSync } from 'node:fs';
```

Comment:
question: It seems odd that we use two functions from this library, but import each function a different way. Shouldn't we stick with one of:

```
import fs from 'fs/promises';
```

or

```
import { existsSync, unlinkSync } from 'node:fs/promises';
```